src/components/accordion-tab/accordion-tab.tsx

```diff
@@ -1,20 +1,20 @@
 import { Component, Prop, h } from '@stencil/core';
 
 @Component({
-  tag: 'accordion-component',
-  styleUrl: 'accordion-component.scss',
+  tag: 'accordion-tab',
+  styleUrl: 'accordion-tab.scss',
   shadow: true,
 })
-export class AccordionComponent {
-  @Prop() title: string;
+export class AccordionTab {
+  @Prop() tabTitle: string;
```

Comment:
I would not change this prop name.
In the scope the accordion tab it is still the title.